dotnet-desktop-guide/net/wpf/data/how-to-bind-to-an-enumeration.md

```diff
@@ -0,0 +1,64 @@
+---
+title: How to bind to an enumeration
+description: Learn how to use data binding to bind an enumeration to a collection object in XAML and in code for Windows Presentation Foundation.
+author: adegeo
+ms.author: adegeo
+ms.date: 04/30/2021
+dev_langs:
+  - "csharp"
+  - "vb"
+helpviewer_keywords: 
+  - "binding data [WPF], enumeration"
+  - "data binding [WPF], enumeration"
+  - "enumeration [WPF]"
+---
+
+# How to bind to an enumeration (WPF .NET)
+
+This example shows how to bind to an enumeration. Unfortunately there isn't a direct way to use an enumeration as a data binding source. However, the <xref:System.Enum.GetValues(System.Type)?displayProperty=nameWithType> method returns a collection of values. These values can be wrapped in a <xref:System.Windows.Data.ObjectDataProvider> and used as a data source.
+
+The <xref:System.Windows.Data.ObjectDataProvider> type provides a convenient way to create an object in XAML and use it as a data source.
+
+[!INCLUDE [desktop guide under construction](../../includes/desktop-guide-preview-note.md)]
+
```

Comment:
Should there be a Prerequisites section?

Reply:
I don't think there is anything special they need. Did you have something in mind?

Reply:
No, just asking.